ijPlugin/src/main/kotlin/org/jetbrains/research/tasktracker/tracking/webcam/WebCamTracker.kt

```diff
@@ -61,6 +59,6 @@ class WebCamTracker(project: Project, private val emoPredictor: EmoPredictor) :
         // 5 sec
         private const val TIME_TO_PHOTO_DELAY = 5000L
 
-        private const val PHOTOS_MADE_BEFORE_NOTIFICATION = 60 * 60 / (TIME_TO_PHOTO_DELAY / 1000)
+        private const val PHOTOS_MADE_BEFORE_NOTIFICATION: Int = (60 * 60 / (TIME_TO_PHOTO_DELAY / 1000)).toInt()
```

Comment:
Are you going to refactor it in another pr? I mean, it is better to implement a service with a coroutine scope to run a delayed showing of a pop up instead of this approach

Reply:
Yeah, I'll resolve it in next PRs